apiserver/Makefile

```diff
@@ -99,8 +99,7 @@ else
 	$(call build_binary, $(PACKAGE_NAME)/cmd/apiserver, $@)
 endif
 else
-	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD) \
-		sh -c '$(GIT_CONFIG_SSH) go build -v -o $@ -ldflags "$(VERSION_FLAGS)" $(PACKAGE_NAME)/cmd/apiserver'
+	$(call build_binary, $(PACKAGE_NAME)/cmd/apiserver, $@)
```

Comment:
Always disable cgo for arches that don't support fips/boringssl expect for binaries that explicitly require cgo.

Reply:
done.